pkg/aws/client.go

```diff
@@ -235,6 +236,46 @@ func (c *awsClient) ValidateCredentials() (bool, error) {
 	return true, nil
 }
 
+// Checks osdCcsAdmin IAM credentials are valid.
+// Get stack's key and actually key and compare them
+// to get the stack credentials:
+// aws cloudformation describe-stack-resource \
+// --logical-resource-id osdCcsAdminAccessKeys --stack-name osdCcsAdminIAMUser
+func (c *awsClient) ValidateCFUserCredentials() error {
+	name := AdminUserName
+	accessKeyInput := &iam.ListAccessKeysInput{
+		UserName: &name,
+	}
+	accessKeyList, err := c.iamClient.ListAccessKeys(accessKeyInput)
+	if err != nil {
+		return err
+	}
+
+	OsdCcsAdminStackNamePtr := OsdCcsAdminStackName
+	LogicalResourceIDPtr := "osdCcsAdminAccessKeys"
+	stackResourceInput := &cloudformation.DescribeStackResourceInput{
+		StackName:         &OsdCcsAdminStackNamePtr,
+		LogicalResourceId: &LogicalResourceIDPtr,
+	}
+	resources, err := c.cfClient.DescribeStackResource(stackResourceInput)
+	if err != nil {
+		return err
+	}
+	cfAccessKey := resources.StackResourceDetail.PhysicalResourceId
+
+	for _, key := range accessKeyList.AccessKeyMetadata {
+		if *key.AccessKeyId == *cfAccessKey && *key.Status == "Active" {
+			return nil
+		}
+	}
+
+	return fmt.Errorf(`
+	Invalid CloudFormation stack credentials: %s is not valid 
+	you can recreate the CloudFormation stack with 
+	"moactl init --delete-stack && moactl init"
+	`, name)
+}
+
```

Comment:
I think this will produce spaces in the error message output. See [here](https://github.com/openshift/moactl/blob/3428aad22789b4155fe34224619a9ed29017e0a5/pkg/aws/client.go#L290) how we solved that problem earlier.

Reply:
thanks, fixed now